nixos/modules/misc/ids.nix

```diff
@@ -306,6 +306,7 @@
       ceph = 288;
       duplicati = 289;
       monetdb = 290;
+      tuntox = 291;
```

Comment:
Are you sure you need the user to be a specific ID? It usually isn't necessary since the system can assign user IDs.